ingestors/misc/jsonfile.py

```diff
@@ -1,6 +1,5 @@
 import json
 
-from followthemoney import model
```

Comment:
Assuming that followthemoney was not being used in this file?

Reply:
Yeah, this is an automatic linter which went too far. We're not using this here, but I shouldn't have changed this out-of-scope file.